packages/core/src/client.test.ts

```diff
@@ -140,7 +140,7 @@ describe('Client', () => {
     window.localStorage.setItem(
```

Comment:
is JWT appropriate for client?  I thought not - but perhaps I'm missing something.

Reply:
Hm, perhaps some naming confusion.

"JWT" - a format of access token, which includes a set of "claims" signed by the server

Medplum uses JWTs for access tokens, even when you're signing in with email/password or google auth or whatever

"JWT bearer grant type" - a special OAuth flow where the client signs their own JWT using the preconfigured client ID and client secret.